cmd/monaco/account/deploy.go

```diff
@@ -132,3 +155,30 @@ func loadAccountManagementResources(fs afero.Fs, projs manifest.ProjectDefinitio
 
 	return resources, errs
 }
+
+func createAccountClients(manifestAccounts map[string]manifest.Account) (map[deployer.AccountInfo]*accounts.Client, error) {
+	accClients := make(map[deployer.AccountInfo]*accounts.Client, len(manifestAccounts))
+	for _, acc := range manifestAccounts {
+		oauthCreds := clientcredentials.Config{
+			ClientID:     acc.OAuth.ClientID.Value.Value(),
+			ClientSecret: acc.OAuth.ClientSecret.Value.Value(),
+			TokenURL:     acc.OAuth.GetTokenEndpointValue(),
+		}
+
+		var apiUrl string
+		if acc.ApiUrl == nil || acc.ApiUrl.Value == "" {
+			apiUrl = "https://api-dev.internal.dynatracelabs.com"
+		} else {
+			apiUrl = acc.ApiUrl.Value
+		}
+		accClient, err := clients.Factory().WithOAuthCredentials(oauthCreds).AccountClient(apiUrl)
```

Comment:
Add the monaco user-agent string to this client please. (I forgot that for delete as well... as it's merged could you maybe create and reuse a helper method for creating the client?)

Reply:
I've already done that today in the morning :) i will use that also in your code

Reply:
this is still open I think?